package.json

```diff
@@ -17,5 +17,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "react-router-dom": "^6.3.0"
```

Comment:
It shouldn't be here !

Reply:
I remove the package in package.json as you ask.